### PR TITLE
Add changelog for v1.5

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add changelog for v1.5.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.5.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Users are now able to use `K8s exec api` or `kubectl exec` command to connect to pods on the edge node. ([#2075](https://github.com/kubeedge/kubeedge/pull/2075), [@daixiang0](https://github.com/daixiang0), [@kadisi](https://github.com/kadisi)).
+
+**Support Keadm Debug Command for Trouble Shooting On Edge Node**
+
+Users are now able to use `keadm debug get/collect` to get/collect data in edgecore.db for trouble shooting, 
+use `keadm debug check/diagnose` to check the running environment on edge. ([#1939](https://github.com/kubeedge/kubeedge/pull/1939), [@shenkonghui](https://github.com/shenkonghui), [@qingchen1203](https://github.com/qingchen1203))
+
+**Kubernetes Dependencies Upgrade**
+
+Upgrade the venderod kubernetes version to v1.19.3, users now can use the feature of new version
```
Comment (chendave) [inline CHANGELOG/CHANGELOG-1.5.md:61]: s/venderod /vendor-ed

File: CHANGELOG/CHANGELOG-1.5.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        * [Bug Fixes](#bug-fixes)
+
+
+# v1.5.0
+
+## Downloads for v1.5.0
+
+### KubeEdge Binaries
+| filename | Size | sha512 hash |
+| -------- | ---- | ----------- |
```
Comment (chendave) [inline CHANGELOG/CHANGELOG-1.5.md:19]: Are you going to fill this column?

Comment (fisherxu) [reply CHANGELOG/CHANGELOG-1.5.md:19]: Will fill it later :)

File: CHANGELOG/CHANGELOG-1.5.md (hunk 1 above)
Comment (fisherxu) [reply CHANGELOG/CHANGELOG-1.5.md]: done

File: CHANGELOG/CHANGELOG-1.5.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+**Mapper Design V2**
+
+New version of Mapper, simplify the function and extract the common code to SDK for Mappers, enhancement include:
+
+- Simplify the structure
+- New Components include: Configmap parser, Driver, Event process, Timer
+- Extract SDK
+
+Users are now able to develop mappers based on the new design standards.([#2147](https://github.com/kubeedge/kubeedge/pull/2147), [@sailorvii](https://github.com/sailorvii), [@luogangyi](https://github.com/luogangyi)).
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.5.md:44]:
```suggestion
**Simplified Device Mapper reference architecture**

New version of Mapper reference architecture:

- Simplified Mapper code structure
- Extracted common code into SDK
- Added new building blocks: Configmap parser, Driver, Event process, Timer

Users are now able to develop mappers based on the new design standard.([#2147](https://github.com/kubeedge/kubeedge/pull/2147), [@sailorvii](https://github.com/sailorvii), [@luogangyi](https://github.com/luogangyi)).
```

File: CHANGELOG/CHANGELOG-1.5.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+**Modbus Mapper Golang Implementation**
+
+A new modbus mapper written in Golang based on new standards above. ([#2282](https://github.com/kubeedge/kubeedge/pull/2282), [@sailorvii](https://github.com/sailorvii)).
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.5.md:48]:
```suggestion
A new modbus mapper with Golang implementation is provided, based on new Device Mapper Standard. ([#2282](https://github.com/kubeedge/kubeedge/pull/2282), [@sailorvii](https://github.com/sailorvii)). 
```

File: CHANGELOG/CHANGELOG-1.5.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+- New Components include: Configmap parser, Driver, Event process, Timer
+- Extract SDK
+
+Users are now able to develop mappers based on the new design standards.([#2147](https://github.com/kubeedge/kubeedge/pull/2147), [@sailorvii](https://github.com/sailorvii), [@luogangyi](https://github.com/luogangyi)).
+
+**Modbus Mapper Golang Implementation**
+
+A new modbus mapper written in Golang based on new standards above. ([#2282](https://github.com/kubeedge/kubeedge/pull/2282), [@sailorvii](https://github.com/sailorvii)). 
+
+**Support Exec To Pod On Edge Node**
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.5.md:50]:
```suggestion
**Support Edge Pod Remote Debugging From Cloud**
```

File: CHANGELOG/CHANGELOG-1.5.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+A new modbus mapper with Golang implementation is provided, based on new Device Mapper Standard. ([#2282](https://github.com/kubeedge/kubeedge/pull/2282), [@sailorvii](https://github.com/sailorvii)). 
+
+**Support Remote Exec to Pods on Edge From Cloud**
+
+Users are now able to use `K8s exec api` or `kubectl exec` command to connect to pods on the edge node. ([#2075](https://github.com/kubeedge/kubeedge/pull/2075), [@daixiang0](https://github.com/daixiang0), [@kadisi](https://github.com/kadisi)).
+
+**Support Keadm Debug Command for Trouble Shooting On Edge Node**
+
+Users are now able to use `keadm debug get/collect` to get/collect data in edgecore.db for trouble shooting, 
+use `keadm debug check/diagnose` to check the running environment on edge. ([#1939](https://github.com/kubeedge/kubeedge/pull/1939), [@shenkonghui](https://github.com/shenkonghui), [@qingchen1203](https://github.com/qingchen1203))
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.5.md:57]:
```suggestion
**Support Keadm Debug Command for Trouble Shooting On Edge Nodes**

A set of keadm debug subcommands are added for Trouble Shooting On Edge Nodes.
Users are now able to use `keadm debug get` and `keadm debug collect` to get/collect KubeEdge local data for trouble shooting, 
and use `keadm debug check` and `keadm debug diagnose` to check local environment configuration. ([#1939](https://github.com/kubeedge/kubeedge/pull/1939), [@shenkonghui](https://github.com/shenkonghui), [@qingchen1203](https://github.com/qingchen1203))
```

File: CHANGELOG/CHANGELOG-1.5.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+- Support config domain URL for cloudcore ([#2126](https://github.com/kubeedge/kubeedge/pull/2126), [@ls889](https://github.com/ls889))
+- Keadm: support arm/arm64 for CentOS ([#2149](https://github.com/kubeedge/kubeedge/pull/2149), [@daixiang0](https://github.com/daixiang0))
+- cloudcore readiness gate show error status ([#2157](https://github.com/kubeedge/kubeedge/pull/2157), [@Yellow-L](https://github.com/Yellow-L))
+- Added the function of unsubscribe topics ([#2188](https://github.com/kubeedge/kubeedge/pull/2188), [@muxuelan](https://github.com/muxuelan))
+- Keadm: add command line option crgoupdriver for subcommand join ([#2202](https://github.com/kubeedge/kubeedge/pull/2202), [@Yellow-L](https://github.com/Yellow-L))
+- Add restart options for cloudcore.service and edgecore.service ([#2207](https://github.com/kubeedge/kubeedge/pull/2207), [@YaozhongZhang](https://github.com/YaozhongZhang))
+- add a option "--package-path" to keadm init/join ([#2213](https://github.com/kubeedge/kubeedge/pull/2213), [@Rachel-Shao](https://github.com/Rachel-Shao))
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.5.md:79]:
```suggestion
- Eventbus add tls config when connect to mqtt ([#2109](https://github.com/kubeedge/kubeedge/pull/2109), [@lvchenggang](https://github.com/lvchenggang))
- Add zero judgment when pods is obtained from the cache ([#2115](https://github.com/kubeedge/kubeedge/pull/2115), [@XiaoJiangWang](https://github.com/XiaoJiangWang))
- Add metrics api support in streamserver ([#2125](https://github.com/kubeedge/kubeedge/pull/2125), [@luogangyi](https://github.com/luogangyi))
- Support config domain URL for cloudcore ([#2126](https://github.com/kubeedge/kubeedge/pull/2126), [@ls889](https://github.com/ls889))
- Keadm: support arm/arm64 for CentOS ([#2149](https://github.com/kubeedge/kubeedge/pull/2149), [@daixiang0](https://github.com/daixiang0))
- Cloudcore readiness gate show error status ([#2157](https://github.com/kubeedge/kubeedge/pull/2157), [@Yellow-L](https://github.com/Yellow-L))
- Added the function of unsubscribe topics ([#2188](https://github.com/kubeedge/kubeedge/pull/2188), [@muxuelan](https://github.com/muxuelan))
- Keadm: add command line option crgoupdriver for subcommand join ([#2202](https://github.com/kubeedge/kubeedge/pull/2202), [@Yellow-L](https://github.com/Yellow-L))
- Add restart options for cloudcore.service and edgecore.service ([#2207](https://github.com/kubeedge/kubeedge/pull/2207), [@YaozhongZhang](https://github.com/YaozhongZhang))
- Add a option "--package-path" to keadm init/join ([#2213](https://github.com/kubeedge/kubeedge/pull/2213), [@Rachel-Shao](https://github.com/Rachel-Shao))
```

File: CHANGELOG/CHANGELOG-1.5.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+- fix deviceProfile json bug ([#2143](https://github.com/kubeedge/kubeedge/pull/2143), [@jidalong](https://github.com/jidalong))
+- fix reDownload serviceFile ([#2170](https://github.com/kubeedge/kubeedge/pull/2170), [@GsssC](https://github.com/GsssC))
+- fix bug: cloud send twin_update confirm message to edge ([#2182](https://github.com/kubeedge/kubeedge/pull/2182), [@jidalong](https://github.com/jidalong))
+- Fix edged log issue ([#2227](https://github.com/kubeedge/kubeedge/pull/2227), [@daixiang0](https://github.com/daixiang0))
+- Fix edgecore service download issue under proxy environment ([#2253](https://github.com/kubeedge/kubeedge/pull/2253), [@llhuii](https://github.com/llhuii))
+- fix missing protocol common config update ([#2265](https://github.com/kubeedge/kubeedge/pull/2265), [@luogangyi](https://github.com/luogangyi))
+- fix a bug of device data type ([#2285](https://github.com/kubeedge/kubeedge/pull/2285), [@wuqihui0317](https://github.com/wuqihui0317))
+- fix cloudhub api /readyz panic ([#2304](https://github.com/kubeedge/kubeedge/pull/2304), [@gccio](https://github.com/gccio))
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.5.md:96]:
```suggestion
- Edged support update pod status after consume added pod ([#2108](https://github.com/kubeedge/kubeedge/pull/2108), [@lvchenggang](https://github.com/lvchenggang))
- Fix resource version compare ([#2120](https://github.com/kubeedge/kubeedge/pull/2120), [@luogangyi](https://github.com/luogangyi))
- Fix/keadm: checksum validation of the downloaded file every time ([#2135](https://github.com/kubeedge/kubeedge/pull/2135), [@ttlv](https://github.com/ttlv))
- Fix deviceProfile json bug ([#2143](https://github.com/kubeedge/kubeedge/pull/2143), [@jidalong](https://github.com/jidalong))
- Fix reDownload serviceFile ([#2170](https://github.com/kubeedge/kubeedge/pull/2170), [@GsssC](https://github.com/GsssC))
- Fix bug: cloud send twin_update confirm message to edge ([#2182](https://github.com/kubeedge/kubeedge/pull/2182), [@jidalong](https://github.com/jidalong))
- Fix edged log issue ([#2227](https://github.com/kubeedge/kubeedge/pull/2227), [@daixiang0](https://github.com/daixiang0))
- Fix edgecore service download issue under proxy environment ([#2253](https://github.com/kubeedge/kubeedge/pull/2253), [@llhuii](https://github.com/llhuii))
- Fix missing protocol common config update ([#2265](https://github.com/kubeedge/kubeedge/pull/2265), [@luogangyi](https://github.com/luogangyi))
- Fix a bug of device data type ([#2285](https://github.com/kubeedge/kubeedge/pull/2285), [@wuqihui0317](https://github.com/wuqihui0317))
- Fix cloudhub api /readyz panic ([#2304](https://github.com/kubeedge/kubeedge/pull/2304), [@gccio](https://github.com/gccio))
```

**Pull request merged**